wcivf/apps/elections/models.py

```diff
@@ -489,3 +489,14 @@ def __str__(self):
     @property
     def uses_party_lists(self):
         return self.slug in ["PR-CL", "AMS"]
+
+    @property
+    def explainer_url(self):
```

Comment:
:+1: for being on the model, but given we don't have a `get_absolute_url` for this model, we could just use that? I think this would mean the admin interface links to it (it will add the "view on site" button).

Also, is returning nothing the best thing to do if we have no match for the voting system? It feels like this is something we should enforce at the DB level (see previous comment), and that having no valid voting system should raise an exception

Reply:
@symroe Are you suggesting to add a get_absolute_url method to this model in place of `explainer_url`? 

👍  on exception

Reply:
Yes I think so - it seems to fit the concept of a unique resource for the model, anyway. I think this is just a case of renaming the method :+1: